index.html

```diff
@@ -0,0 +1,94 @@
+                            />
+                          </svg>
+                        </span>
+                      </li>
+                      <li class="h-nav-item"><span>Store</span></li>
+                      <li class="h-nav-item"><span>Mac</span></li>
+                      <li class="h-nav-item"><span>iPod</span></li>
+                      <li class="h-nav-item"><span>iPhone</span></li>
+                      <li class="h-nav-item"><span>iPad</span></li>
+                      <li class="h-nav-item"><span>iTunes</span></li>
+                      <li class="h-nav-item"><span>Support</span></li>
+                      <li class="h-nav-item">
+                        <input type="text" />
+                        <svg class="svg-search" width="15px" viewBox="0 0 16 16">
+                          <path
+                            fill="#eee"
+                            d="M11.3,9.2C11.7,8.4,12,7.5,12,6.5C12,3.5,9.5,1,6.5,1S1,3.5,1,6.5S3.5,12,6.5,12c1,0,1.9-0.3,2.7-0.7l3.3,3.3c0.3,0.3,0.7,0.4,1.1,0.4s0.8-0.1,1.1-0.4c0.6-0.6,0.6-1.5,0-2.1L11.3,9.2zM6.5,10.3c-2.1,0-3.8-1.7-3.8-3.8c0-2.1,1.7-3.8,3.8-3.8c2.1,0,3.8,1.7,3.8,3.8C10.3,8.6,8.6,10.3,6.5,10.3z"
+                          ></path>
+                        </svg>
+                      </li>
+                    </ul>
+                  </nav>
+                  <div class="headings">
+                    <h1>What will your verse be?</h1>
+                    <span>Everyone has something to share. See how people are using <br> iPad in their unique ways to add to the world’s story.</span>
+                    <a href=""><p>Watch the “Your Verse” film and explore the stories</p></a>
+                </div>
+
+        </div>
+       
+
+    </div>
+    <aside>
+        <div class="promo-images">
+            <img src="assests/images/promo_iphone5s.jpg" alt="Iphone 5s">
+            <img src="assests/images/promo_iphone5c.jpg" alt="iphone 5c">
+            <img src="assests/images/promo_30_years.jpg" alt="promo">
+            <img src="assests/images/promo_supplier_responsibility.jpg" alt="supplier">
+        </div>
```

Comment:
These images are links so you should wrap them in an a tag and do well to make the links dead too